CHANGELOG.md

```diff
@@ -0,0 +1,689 @@
+## 1.30.0
+
+#### New features
+
+* **Allow users to configure apply_prep plan function** ([#1123](https://github.com/puppetlabs/bolt/issues/1123))
```

Comment:
Some other things to consider for this week's release notes
1. I think we should mention https://github.com/puppetlabs/bolt/issues/1172
2. Is this a good place to warn we will be deprecating ubuntu 1404
3. We dont have GH issues for the task helper bugs or the change in action duration format with the human outputter, do we need/want to highlight those?
4. I think we can easily get https://github.com/puppetlabs/bolt/issues/1148 in this release which would need a release note

Reply:
I assume non-issued tickets are maintenance work, and don't need release notes. The duration formate seems like maintenance, the task_helper bugs seem like they should be included. We can still include a release note with either a link to the PR or just no link.

Reply:
@donoghuc  I wrote up this release note for #1172. Look OK?

>>>`bolt-inventory-pdb` was not installed on path (1.30.0)
During Bolt installation, the `bolt-inventory-pdb` tool is now installed on the user's path. This lets the user run it from the command line without knowing the exact location of the tool.